### ISSUE TITLE
Create Sxxxx (`jsx-a11y/anchor-has-content`): Anchors should have content

### ISSUE THREAD

**Issue by ilia-kebets-sonarsource** [SonarSource/SonarJS#4279]
[plugin rule](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/anchor-has-content.md#jsx-a11yanchor-has-content)
